extensions/interactions/base_test.py

```diff
@@ -355,80 +355,75 @@ def test_default_interactions_are_valid(self):
         all_interaction_ids = (
             interaction_registry.Registry.get_all_interaction_ids())
         for interaction_id in all_interaction_ids:
+            # TODO(#10212):Ignoring the RatioExpressionInput for now as
+            # there's no learner's flow for the interaction yet.
+            # Once added this needs to be removed.
```

Comment:
Please be specific to what needs to be removed (since we certainly don't want to remove the whole test here).

Reply:
there would be a bunch of files that need to be taken care of should I specify all of them? it would be roughly 6 - 8 files

Reply:
Yes. The instructions in a TODO should be fully explicit.

In addition, the TODO is in the wrong place. Put it at the places which need to be removed. Make it so that the person who is sorting out the TODO has to do as little work as possible to figure out what is going on.

Finally, phrase your TODO as an instruction. "Remove ... as soon as ...". You don't need all the preamble, just state the action to be done, and the trigger for when it can be done.

Reply:
got it

Reply:
Updated @seanlip . Let me know if there's anything missing.